### PR TITLE
Reactive

### PR DESCRIPTION
**Pull request by kermakovich**

### CODE REVIEW

**Comment by kermakovich** [comment]
Open Feign was removed, because it doesn't support spring boot v. 3. Webflux is used instead

File: src/main/java/solvd/laba/ermakovich/hf/web/WebClientConfig.java [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+ * @author Ermakovich Kseniya
+ */
+@Configuration
+public class WebClientConfig {
+
+    @Bean
+    @LoadBalanced
+    public WebClient.Builder webClient() {
+        return WebClient.builder();
+//                .defaultHeader(HttpHeaders.CONTENT_TYPE, MediaType.APPLICATION_JSON_VALUE)
```
Comment (h1alexbel) [inline src/main/java/solvd/laba/ermakovich/hf/web/WebClientConfig.java]: remove the comments

**Review by h1alexbel — CHANGES_REQUESTED** [review]

**Comment by h1alexbel** [comment]
@kermakovich can you resolve merge conflict?

**Comment by kermakovich** [comment]
@h1alexbel, conflict was resolved

**Comment by h1alexbel** [comment]
@kermakovich thanks

**Pull request merged**